src/plugins/vis_type_table/public/components/table_vis_component.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useState, useCallback, useMemo } from 'react';
+import { EuiDataGridProps, EuiDataGrid } from '@elastic/eui';
```

Comment:
Can't we import and use the OUI named components instead?

Reply:
I did use OUI before but I got this:
<img width="1597" alt="Screen Shot 2022-09-22 at 10 55 45" src="https://user-images.githubusercontent.com/79961084/191818226-21b1c5ee-4ab6-415a-ae37-3d6d1584f663.png">

If I change to EUI then I could render a rough data grid.
<img width="1595" alt="Screen Shot 2022-09-18 at 12 27 31" src="https://user-images.githubusercontent.com/79961084/191818335-8f669a7f-6520-41b8-8fcd-cd351e4ea812.png">

I will open an issue both in our repo and OUI repo. To move forward, I will just use EUI for now.

Reply:
from @AMoo-Miki:
```
i think it is because of how OSD is importing oui. for now OSD should continue to use EUI until we are sure of OUI components don't add weight to the js
```